library/src/main/java/com/airbnb/android/airmapview/AirMapView.java

```diff
@@ -80,6 +80,18 @@ public void initialize(FragmentManager fragmentManager, AirMapInterface mapInter
    * @param fragmentManager required for initialization
    */
   public void initialize(FragmentManager fragmentManager) {
+    initialize(fragmentManager, null, null);
+  }
+
+  /**
+   * Used for initialization of the underlying map provider. Uses the default preferred map provider
+   * (currently Native Google Maps, then Mapbox Web maps, then Web Google Maps).
+   *
+   * @param fragmentManager required for initialization
+   * @param mapboxAccessToken Mapbox Access Token (required for Mapbox maps)
+   * @param mapboxMapId Mapbox Map Id (required for Mapbox maps)
+   */
+  public void initialize(FragmentManager fragmentManager, String mapboxAccessToken, String mapboxMapId) {
```

Comment:
If we go with this solution, then we should remove the setters for mapbox token and app ID so that the user is required to set them in the `AndroidManifest`.